lib/experimental/Navigation/Page/index.stories.tsx

```diff
@@ -0,0 +1,75 @@
+import { Placeholder } from "@/lib/storybook-utils"
+import type { Meta, StoryObj } from "@storybook/react"
+import { Page } from "."
+
+import Header from "../Header/Header"
+import * as HeaderStories from "../Header/Header/index.stories"
+import { Tabs } from "../Tabs"
+import * as TabsStories from "../Tabs/index.stories"
+
+import { StandardLayout } from "@/experimental/Layouts/StandardLayout"
+
+const meta: Meta<typeof Page> = {
+  component: Page,
+  tags: ["autodocs"],
+  parameters: {
+    layout: "fullscreen",
+  },
+  decorators: [
+    (Story) => (
+      <div className="bg-f1-background-tertiary p-2">
+        <Story />
+      </div>
+    ),
+  ],
+}
+
+export default meta
+type Story = StoryObj
+
+export const Default: Story = {
+  args: {
+    header: (
+      <>
+        <Header
+          {...HeaderStories.FirstLevel.args}
+          menu={{ show: false, onClick: () => {} }}
+        />
+        <Tabs {...TabsStories.Primary.args} />
+      </>
+    ),
+    children: (
+      <StandardLayout>
+        {Array(25)
+          .fill(0)
+          .map((_, index) => (
+            <Placeholder key={index} className="min-h-24" />
+          ))}
+      </StandardLayout>
+    ),
+  },
+}
+
+export const FullWidth: Story = {
+  args: {
+    header: (
+      <>
+        <Header
+          {...HeaderStories.FirstLevel.args}
+          menu={{ show: false, onClick: () => {} }}
+        />
+        <Tabs {...TabsStories.Primary.args} />
+      </>
+    ),
+    layout: "fullWidth",
```

Comment:
This is no longer used right?

Reply:
Oops! You're right, that entire story is no longer used!